tests/mmptest/src/AOTTests.cs

```diff
@@ -34,7 +34,7 @@ void ValidateAOTStatus (string tmpDir, Func<FileInfo, bool> shouldAOT, string bu
 		// AOT unit tests can be found in tools/mmp/tests
 		[Test]
 		public void AOT_SmokeTest () {
-			RunMMPTest (tmpDir => {
+			MMPTests.RunMMPTest (tmpDir => {
```

Comment:
why not move `RunMMPTest` to a new, base class ? that way you won't have to prefix `MMPTests` everywhere :)

Reply:
Not a bad idea. Next pass at make-mmp-tests-less-embarrassing will include that.